proto/token-tracker.proto

```diff
@@ -0,0 +1,54 @@
+syntax = "proto3";
+
+package token_tracker;
+
+enum AirTokenStandard{
+    UNKNOWN = 0;
+    ERC20 = 1;
+    ERC721 = 2;
+    ERC1155 = 3;
+}
+
+message AirTransfers{
+    repeated AirTransfer air_transfers = 1;
+    repeated AirERC1155TransferBatch air_transfer_batchs = 2;
+    repeated AirERC1155TransferSingle air_transfer_singles = 3;
+}
+
+message AirTransfer{
+    string hash = 1;
```

Comment:
tokenAddress is missing in all three.

Reply:
Should I keep separate token address and token id for erc1155 as well??

Reply:
Yes tokenAddress is contract address & tokenId id nft id